spec/models/movie.spec.js

```diff
@@ -0,0 +1,44 @@
+var app = require("../../app");
+var db = app.get("db");
+
+var Movie = require('../../models/movie')
+
+describe('Movie', function () {
+  var title1 = "The Exorcist";
+  var overview1 = "12-year-old Regan MacNeil begins to adapt an explicit new personality as strange events befall the local area of Georgetown. Her mother becomes torn between science and superstition in a desperate bid to save her daughter, and ultimately turns to her last hope: Father Damien Karras, a troubled priest who is struggling with his own faith.";
+  var inventory1 = 7;
+
+  afterEach(function () {
+    db.end()
+  })
+
+  describe('#all', function () {
+    it('should return a list of all the movies and respective data', function (done) {
```

Comment:
Is this test testing that the respective movie data is being returned?  It's definitely testing that a list of movies is being generated.  Is there a test you could write that would show that, say, `overview1` is being returned as well?